### PR TITLE
[NO QA] ignore "coverage" folder generated by Jest

### PR DESCRIPTION
**Pull request by marcochavezf**
### Details

This change ignores in git the `coverage` folder generated by Jest:
Screen Shot 2022-01-13 at 15 17 13

### Fixed Issues

N/A

### Tests

Run `npm run test -- --coverage` and verify the `coverage` folder is created and its files are ignored by git:

Screen Shot 2022-01-13 at 15 23 19

Screen Shot 2022-01-13 at 15 23 27

### QA Steps

N/A

### Tested On

- [ ] Web
- [ ] Mobile Web
- [ ] Desktop
- [ ] iOS
- [ ] Android

### Screenshots

N/A

### CODE REVIEW

**Review by thienlnam — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 1]
```diff
@@ -77,4 +77,7 @@ storybook-static
 
 # GitHub GPG Keys
 .github/workflows/OSBotify-private-key.asc
-*.asc
\ No newline at end of file
+*.asc
+
+# Jest coverage report
+coverage/
```
[the remark this answers is not in the document]
Comment (marcochavezf) [reply .gitignore:83]: Hi @thienlnam! are you referring to a new line at the end of the file, right? Also, just curious, why we should ignore `/coverage.data` too?

[the remark this answers is not in the document]
Comment (thienlnam) [reply .gitignore:83]: Ah nvm, was checking other instances of gitignore using jest but it doesn't seem that we have that anyways.  Will merge

[the remark this answers is not in the document]
Comment (thienlnam) [reply .gitignore:83]: We still need a new line at the end of the file though

[the remark this answers is not in the document]
Comment (marcochavezf) [reply .gitignore:83]: Oh I see, I'm going to add it, probably there are different files/data included when the report format is changed.

**Comment by marcochavezf** [comment]
Updated with the suggested changes

Screen Shot 2022-01-18 at 15 18 41

**Review by thienlnam — APPROVED** [review]

**Pull request merged**